scripts/utils.py

```diff
@@ -67,34 +73,50 @@ def load_mesh_mapping_file(mesh_mapping_file_path) -> dict:
   with open(mesh_mapping_file_path, 'r') as f:
     return json.load(f)
 
-def map_mesh_to_values(mesh_code, loaded_mesh_mapping) -> list:
+def map_mesh_to_values(mesh_code, loaded_mesh_mapping):
```

Comment:
In `smpc-global` we have a file called `meshTerms.json`. There you can access by mesh code in O(1). No need to search the mesh file.